Tests/IGListStackSectionControllerTests.m

```diff
@@ -85,8 +85,9 @@ - (void)test_whenInitializingStack_thatNumberOfItemsMatches {
     section4.items = 1;
 
     IGListStackedSectionController *stack = [[IGListStackedSectionController alloc] initWithSectionControllers:@[section1, section2, section3, section4]];
-
-    XCTAssertEqual([stack numberOfItems], 6);
+    [stack performBatchAnimated:false updates:^{} completion:^(BOOL finished) {
+        XCTAssertEqual([stack numberOfItems], 6);
```

Comment:
@jeffbailey oh! I missed that. Good call.

Instead of doing the `performSelector` hack, how about we expose `reloadData` in the internal header and call it directly?

https://github.com/Instagram/IGListKit/blob/master/Source/Internal/IGListStackedSectionControllerInternal.h

The header is already imported to the test.